### PR TITLE
Start implementing `smz o` for orphans

### PR DESCRIPTION
**Pull request by pengwynn**
This PR is for the branch @evantravers started to explore a way to list all orphans, or notes without backlinks.

### CODE REVIEW

File: smz [hunk 1]
```diff
@@ -55,6 +56,15 @@ sub_backlinks(){
     search_in_files "$search_pattern" "$SMZPATH" --files-with-matches | grep -v "$1"
 }
 
+sub_orphans(){
+    for zettel in "$SMZPATH"/*
+    do
+      if [ -z "$(smz b "$zettel")" ]; then
```
Comment (pengwynn) [inline smz:62]: There's probably a way to do this and avoid the subshell, but this seems to work just fine.

**Comment by pengwynn** [comment]
@evantravers can you see if you still get the warning about the `\n`? I stopped getting it after the changes in this diff.

**Comment by evantravers** [comment]
I'm still getting it…

```
Consider enabling multiline mode with the --multiline flag (or -U for short).
When multiline mode is enabled, new line characters can be matched.
/Users/evan/Dropbox/wiki/archive//20200315091640-thoughts-on-covid19-and-social-distancing.md
the literal '"\n"' is not allowed in a regex
```

**Comment by pengwynn** [comment]
@evantravers what do you have your `$SMZPATH` set to? It's strange to see a double slash in the path.

**Comment by evantravers** [comment]
@pengwynn sorry… I have two computers, and one of them had a trailing slash on `$SMZPATH`. That's been repaired, but I'm still getting the bug. 

Possibly related: at the moment, I'm getting this error after _every_ `smz` command:
```
~/Dropbox/wiki/notes/bible_study
❯ smz f owner | pbcopy
realpath: '': No such file or directory
```

Errors I'm currently seeing after `smz o`:

```
/Users/evantravers/Dropbox/wiki/archive/20200405203512-the-summarization-of-journaling-is-like-teaching.md
realpath: '': No such file or directory
/Users/evantravers/Dropbox/wiki/archive/20200405220629-growth-mindset-is-compatible-with-extreme-ownership.md
realpath: '': No such file or directory
the literal '"\n"' is not allowed in a regex

Consider enabling multiline mode with the --multiline flag (or -U for short).
When multiline mode is enabled, new line characters can be matched.
```

Here's one of the files that the error we were shows up after…

/Users/evantravers/Dropbox/wiki/archive/20200405220629-growth-mindset-is-compatible-with-extreme-ownership.md
```
tags: #mindset, #pride, #ego, #leadership
id: 20200405220629

Growth Mindset is compatible with Extreme Ownership

> at critical decision points, they opted for what would make them feel good and look good over what would serve the longer-term corporate goals.

(Dweck, 122)[^2]

Jocko warns about this constantly in his book.[^1] You have to drop ego first. Mission-focused. 

[^1]: Extreme Ownership
[^2]: Carol S. Dweck. _Mindset_. Random House, 2006. [[20190617191608]]
```

File: smz [hunk 2]
```diff
@@ -13,14 +13,15 @@ if [ ! -d "$SMZPATH" ]; then
 fi
 
 if [ -x "$(command -v realpath)" ]; then
-  ZKPATH=$(realpath --relative-to="$PWD" "$ZKPATH")
+  SMZPATH=$(realpath --relative-to="$PWD" "$SMZPATH")
 fi
```
Comment (pengwynn) [inline smz:17]: Good catch. 👏

**Pull request closed without merging**